src/Relation/Binary/ExtensionalEquivalence.agda

```diff
@@ -0,0 +1,37 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Definition of extensional equivalence.
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+open import Level
+open import Relation.Binary.Bundles
+open import Relation.Binary.Definitions
+
+module Relation.Binary.ExtensionalEquivalence
+  {b ℓ : Level}
+  (B-setoid : Setoid b ℓ)
+  where
+
+private
+  variable
+    a : Level
+    A : Set a
+
+open Setoid B-setoid renaming (Carrier to B)
+
+infix 4 _≗_
+
+_≗_ : ∀ {a} → {A : Set a} → (f g : A → B) → Set (ℓ ⊔ a)
+f ≗ g = ∀ x → f x ≈ g x
+
+≗-refl : ∀ {a} → {A : Set a} → Reflexive (_≗_ {A = A})
```

Comment:
You can avoid a set of brackets by writing `Reflexive {A = A} _≗_`

Reply:
Ah sorry, yes sorry, should be something like `Reflexive {A = A → B} _≗_`?

Reply:
Yep, that worked; thanks!